zerver/lib/markdown/fenced_code.py

```diff
@@ -549,7 +549,7 @@ def makeExtension(*args: Any, **kwargs: None) -> FencedCodeExtension:
     return FencedCodeExtension(kwargs)
 
 
-if __name__ == "__main__":
+if __name__ == "__main__":  # nocoverage - can we remove this?
```

Comment:
We can probably remove this -- this file was forked from upstream Python-markdown, and this block of code come from the original there ... but of course this block won't run inside our codebase.